src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -149,6 +168,20 @@ def skip_schema(self, schema: str):
         except BadRequest as err:
             logger.error(err)
 
+    def unskip_schema(self, schema: str):
+        # Removes skip mark from the schema property
+        try:
+            self._sql_backend.execute(
+                f"ALTER SCHEMA {escape_sql_identifier(schema)} UNSET DBPROPERTIES IF EXISTS('{self.UCX_SKIP_PROPERTY}');"
+            )
+        except NotFound as err:
+            if "[SCHEMA_NOT_FOUND]" in str(err):
+                logger.error(f"Failed to remove skip marker for Schema {schema}. Schema not found.")
```

Comment:
```suggestion
                logger.error(f"Failed to remove skip marker from schema: {schema}", exc_info=e)
```

Reply:
Or if you prefer to hide the error:

```suggestion
                logger.error(f"Schema not found: {schema}")
```